source/tutorials/nix-language.md

```diff
@@ -213,17 +213,17 @@ nix-instantiate --eval --strict file.nix
 
 You may quickly encounter Nix language expressions that look very complicated.
 
-As with any programming language, the required amount of Nix language code closely matches the complexity of the problem it is supposed to solve, and reflects how well the problem – and its solution – is understood.
+As with any programming language, the required amount of Nix language code closely matches the complexity of the problem it is supposed to solve and reflects how well the problem – and its solution – is understood.
 
 Building software is a complex undertaking, and Nix both *exposes* and *allows managing* this complexity with the Nix language.
 
-The purpose of the Nix language is to create *build tasks*: precise descriptions of how contents of existing files are used to derive new files.
+The purpose of the Nix language is to create *build tasks*: precise descriptions of how the contents of existing files are used to derive new files.
 
 :::{important}
 A build task in Nix is called a *derivation*.
 :::
 
-The Nix language has only few basic constructs which can be combined arbitrarily:
+The Nix language has a few basic constructs which can be combined arbitrarily:
```

Comment:
```suggestion
The Nix language has a small number of basic constructs which can be combined arbitrarily:
```

Does that convey the meaning more precisely? I want to emphasize  "there are not many of them", not colloquially "there are only a couple of them".